packages/react-components/src/molecules/EventInfo.tsx

```diff
@@ -98,18 +98,17 @@ const EventInfo: React.FC<EventInfoProps> = ({
           </Headline3>
         </Anchor>
         <EventTime {...props} />
-        {groups.length ? (
+        {group ? (
           <ul css={groupsStyles}>
-            {groups.map((group) => (
-              <li key={group.id} css={eventOwnerStyles}>
-                <Link
-                  href={network({}).groups({}).group({ groupId: group.id }).$}
-                >
-                  <span css={iconStyles}>{groupsIcon}</span>
-                  {group.name}
-                </Link>
-              </li>
-            ))}
+            <li key={group.id} css={eventOwnerStyles}>
```

Comment:
no longer a list. can now probably even merge it into the `div` below and make only styles in it conditional